City-of-Helsinki/youth-membership-ui
# YM-412 | Approval view

## Description

Improved texts that were shown to user when something went wrong and an error occurred (approval view). I also added this error code to the ignore list.

## Context


This should improve user experience, no random error popups when user did nothing wrong.

[YM-412](https://helsinkisolutionoffice.atlassian.net/browse/YM-412)

## How Has This Been Tested?

Manually + normal tests. Didn't run `e2e` - tests, our mailosaur trial has ended and some of the tests are no longer working... (I'm working to resolve this issue)

## Manual Testing Instructions for Reviewers

- Create new youth profile (age < 18). 
- Follow the email (remember to replace url before `/approve/` with `localhost` and approve profile
- After profile is approved and message is displayed, refresh the page or enter the page again vie email link
- You should see more descriptive message
- You should not see error message on the right side.
